image-layout.md

```diff
@@ -156,4 +161,48 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## index.json
+
+This OPTIONAL file is for use indexing references and elements of the image-layout, where walking the `refs` and `blobs` is sub-optimal.
```

Comment:
> I didn't do that because it was not my intention to anihilate the `./refs/` directory.

Define something like `org.opencontainers.ref.name` lets consumers use the index to access content by name (e.g. `oci-unpack --ref v1.0 foo/ bar/`) vs. having callers access it by annotation (e.g. `oci-unpack --annotation org.opencontainers.ref.name=v1.0 foo/ bar/`).  But being able to use the index to access content by name doesn't mean you have to drop `refs/` (which is another way to access content by name).